goodtablesio/config.py

```diff
@@ -1,13 +1,21 @@
 import os
+import logging
 from dotenv import load_dotenv
 load_dotenv('.env')
 
 
+# General
+
+TABULAR_EXTENSIONS = ['csv', 'xls', 'xlsx', 'ods']
```

Comment:
Yes, that's me. I thought about asking @roll where to get these from but forgot about it.